packages/admin/cms-admin/src/blocks/DamVideoBlock.tsx

```diff
@@ -116,15 +119,31 @@ export const DamVideoBlock: BlockInterface<DamVideoBlockData, State, DamVideoBlo
                     label={<FormattedMessage id="comet.blocks.video.autoplay" defaultMessage="Autoplay" />}
                     component={FinalFormSwitch}
                 />
+                <Field
+                    type="checkbox"
+                    name="loop"
+                    label={<FormattedMessage id="comet.blocks.video.loop" defaultMessage="Loop" />}
+                    component={FinalFormSwitch}
+                />
                 <Field
                     type="checkbox"
                     name="showControls"
                     label={<FormattedMessage id="comet.blocks.video.showControls" defaultMessage="Show controls" />}
                     component={FinalFormSwitch}
+                    validate={(v) => {
+                        if (!(state.autoplay || state.showControls)) {
+                            return intl.formatMessage({
```

Comment:
You may use `FormattedMessage` here directly, which removes the extra import of the hook